docs/query-builder.txt

```diff
@@ -1033,6 +1034,30 @@ to specify a maximum duration to wait for cursor operations to complete.
    `MongoDB\Collection::find() <https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBCollection-find/>`__
    in the PHP Library documentation.
 
+.. _laravel-query-builder-read-pref:
+
+Specify a Read Preference Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can control how the {+odm-short+} directs read operations to replica
+set members by setting a read preference. 
+
+The following example queries the ``movies`` collection for documents
+in which the ``runtime`` value is greater than ``240``. The example passes a
+value of ``"secondary"`` to the ``readPreference()`` method, which sends 
+the query to secondary replica set members:
```

Comment:
I updated the example to use the constant [`MongoDB\Driver\ReadPreference::SECONDARY_PREFERRED`](https://www.php.net/manual/en/class.mongodb-driver-readpreference.php#mongodb-driver-readpreference.constants.secondary-preferred), this description need to be updated accordingly.